### ISSUE TITLE
Disallow user python site-packages

### ISSUE THREAD

**Issue by dotfloat** [equinor/komodo#136]
Komodo uses a vanilla Python interpreter, which by defaults adds `~/.local/lib/pythonX.Y/site-packges` to `sys.path`, overriding stuff from Komodo. This will cause us headaches, such as user `pip install --user` a package, never updates it. We add this package to Komodo, user complains when everything breaks.

This already broke `komodoenv` for me, because I have an incompatible version of `virtualenv` in my `.local/lib/pythonX.Y/site-packages`. If any users expect Komodo to work like this, then we should inform them that they are doing it wrong.

Jonas' virtualenv approach already has breaking changes wrt. this: `virtualenv` explicitly does not add user site-packages. Depending on how long we'll keep shipping our own python binaries, we should consider disabling user site-packages via compile-flags.

Tagging @markusdregi @jondequinor and then running away to :beach_umbrella: .

**Comment by jondequinor** [comment]
I agree. But I cannot find any relevant compile flags. What did you have in mind?

`python -s` disables it runtime, but I can't find any compile time options.
